agent/assets/agent-create-registries-conf-playbook.yaml

```diff
@@ -0,0 +1,19 @@
+- name: Create registries.conf for Agent Installer
+  hosts: localhost
+  collections:
+   - community.general
+  gather_facts: no
+  vars:
+    - mirror_images: "{{ lookup('env', 'MIRROR_IMAGES') }}"
+    - oc_mirror: "{{ lookup('env', 'OC_MIRROR') }}"
+    - mirror_path: "{{ lookup('env', 'MIRROR_PATH') }}"
```

Comment:
Looks like that `MIRROR_PATH` needs to be exported

Reply:
Will address in a follow up PR